komga-webui/src/views/BookReader.vue

```diff
@@ -533,6 +544,10 @@ export default Vue.extend({
           page: this.currentPage.toString()
         }
       })
+      this.updateTitle()
```

Comment:
will be called way too often, as `updateRoute` is called on each page change

Reply:
This is fair, I had messed around with having the current page number in the document.title, which was why it was updating every route update. 
This title is only determined the first time anyway, so I'll remove it from updateRoute